### PR TITLE
update badges and add lifecycle

### LINKED ISSUE #10: Add lifecycle badge

### PR DESCRIPTION
**Pull request by joethorley**
closes #10

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,5 +10,5 @@
 
 *.html
 
-docs/
 inst/docs/
+docs
```
Comment (krlmlr) [inline .gitignore]: To be very precise:

```suggestion
/inst/docs/
/docs/
```

File: .github/workflows/test-coverage.yaml [hunk 2]
```diff
@@ -15,36 +17,45 @@ jobs:
...
             clean = FALSE,
-            install_path = file.path(Sys.getenv("RUNNER_TEMP"), "package")
+            install_path = file.path(normalizePath(Sys.getenv("RUNNER_TEMP"), winslash = "/"), "package")
           )
+          covr::to_cobertura(cov)
         shell: Rscript {0}
 
+      - uses: codecov/codecov-action@v4
+        with:
+          fail_ci_if_error: ${{ github.event_name != 'pull_request' && true || false }}
```
Comment (krlmlr) [inline .github/workflows/test-coverage.yaml]: Do we change that?

File: tests/testthat/test-xnew-value.R [hunk 3]
```diff
@@ -1,8 +1,9 @@
 test_that("xnew_value", {
   data <- tibble::tibble(
     a = c(1L, 3L, 4L),
-    b = c(4,4.5,6),
-    d = c("a", "b", "c"))
+    b = c(4, 4.5, 6),
+    d = c("a", "b", "c")
```
Comment (krlmlr) [inline tests/testthat/test-xnew-value.R]: My pet peeve, here and elsewhere:

```suggestion
    d = c("a", "b", "c"),
```

**Review by krlmlr — APPROVED** [review]
Nice!

**Review by aylapear — APPROVED** [review]

**Pull request merged**